src/functionalTest/java/uk/gov/hmcts/reform/iacaseapi/CcdScenarioRunnerTest.java

```diff
@@ -335,6 +335,12 @@ private Headers getAuthorizationHeaders(Map<String, Object> scenario) {
                 .getHomeOfficeGenericAuthorization();
         }
 
+        if ("JudgeX".equalsIgnoreCase(credentials)) {
```

Comment:
This can just be Judge representing the role to be inline with other roles we have than JudgeX. It's also done as a part of RIA-1280 PR.